scripts/check-tags.ts

```diff
@@ -38,16 +49,24 @@ const checkProjects = async () => {
   const projects = globSync('{apps,libs}/**/project.json', {
     ignore: '**/node_modules/**',
   })
-  let hasError = false
+  let invalidProjects = []
   for (const project of projects) {
-    hasError = hasError || (await checkTags(project))
+    // console.log("Checking project", project)
+    // Old
+    // hasError = hasError || (await checkTags(project))
+
+    // New, and faster
```

Comment:
How is this faster?

Reply:
It's no longer faster. In a previous iteration I early-exited using `break`. Now I'm checking all projects for nicer outputs.